src/main/java/lotto/domain/ManualNumberOfTicket.java

```diff
@@ -0,0 +1,52 @@
+package lotto.domain;
+
+import lotto.utils.ConvertUtil;
+
+import java.util.Objects;
+
+public class ManualNumberOfTicket {
+    private static final String CHECK_MANUAL_NUMBER_OF_TICKET = "입력하신 구매 티켓 수 %s가 숫자인지 확인해주세요.";
+    private static final String CHECK_BOUND = "입력하신 구매 티켓 수가 구매 가능한 티켓 수보다 많습니다.";
+    private static final String CHECK_POSITIVE = "입력하신 구매 티켓 수가 음수입니다.";
+    private static final int MIN = 0;
+    private final int manualNumberOfTicket;
+
+    public ManualNumberOfTicket(String manualNumberOfTicket, TotalNumberOfTicket totalNumberOfTicket) {
+        this(ConvertUtil.toIntOrThrow(manualNumberOfTicket, () -> new IllegalArgumentException(String.format(CHECK_MANUAL_NUMBER_OF_TICKET, manualNumberOfTicket))), totalNumberOfTicket);
```

Comment:
```suggestion
        this(toInt(manualNumberOfTicket), totalNumberOfTicket);
```
지금은 가독성이 너무 떨어지는것 같아요 ㅜㅜ 
`private static int toInt` 같은 변환함수를 내부에 만들어 가독성을 올려봅시다.

Reply:
해해당 부분 이외에 `ConvertUtil.toIntOrThrow` 사용하는 부분들도 내부에 `private static int toInt` 만들어 가독성을 올려주었습니다. 하지만, 각 객체마다 `private static int toInt`라는 중복 함수를 가지게 되는 문제점이 생겨 해당 함수를 `ConverUtil`로 옮겨주었습니다.

`ConvertUtil.toIntOrThrow`을 생성하게 된 계기는 커스텀 오류 출력문을 위해서 였는데 
지금 생각해보면 정수로 변환시키는 과정에 있어서 `Integer.parseInt()`를 사용했더라면, 가독성 뿐만아니라 중복코드들이 생성되는 문제점도 발생하지 않고 결국에는 `ConverUtil`을 만들지도 않아도 됐을 것 같고, 타입 변환을 위해서 이렇게 유틸성으로 만드는 방법이 좋은 방법이 아닐 수도 있다라는 생각이 드네요ㅠㅠ

Reply:
맞습니다. ConvertUtil 이 필요 없어 보이네요. 저도 방향을 잘못잡았다는 생각이 드는데 `ConvertUtil` 에 대한 피드백 핵심은 도메인 관련 용어, 클래스, 예외등이 유틸클래스에 존재하지 않게 하라는 부분이었습니다.ㅎㅎ

Reply:
유틸 클래스에 대한 피드백을 제대로 이해하지 못한 채 수정을 하면서 돌고 돌아 말씀하신 의도를 조금이나마 알게 된 것 같습니다ㅠㅠ
이와 같이 겪은 과정이 앞으로 유틸성 클래스를 만들 때 도움이 될 것 같습니다! 감사합니다:)